89.md

```diff
@@ -0,0 +1,62 @@
+NIP-89
+======
+
+Bitcoin Transaction Broadcasting Events
+-------------------------
+
+`draft` `optional` `author:benthecarman`
+
+A special event with kind `28333`, that contains an encoded bitcoin transaction.
+The bitcoin transaction is encoded as a base64 string, and is contained in the `content` field.
+
+The network magic is contained in the `tags` field. This is to identify the network that the transaction is for.
+The common ones are:
+
+- mainnet: f9beb4d9
```

Comment:
Slight nit:

| Network        | Magic value | Sent over wire as |
|----------------|-------------|-------------------|
| main           | 0xD9B4BEF9  | F9 BE B4 D9        |
| testnet/regtest| 0xDAB5BFFA  | FA BF B5 DA        |
| testnet3       | 0x0709110B  | 0B 11 09 07        |
| signet(default)| 0x40CF030A  | 0A 03 CF 40        |

https://en.bitcoin.it/wiki/Protocol_documentation#Common_structures

There are so far 3 testnets : perhaps align with the descriptions above ie testnet3 instead of testnet

Reply:
updated testnet to testnet3, otherwise I think this is enough information for the user to understand